docs/extensiondev.rst

```diff
@@ -27,13 +27,10 @@ The name of the actual extension (the human readable name) however would
 be something like "Flask-SimpleXML".  Make sure to include the name
 "Flask" somewhere in that name and that you check the capitalization.
 This is how users can then register dependencies to your extension in
-their :file:`setup.py` files.
-
-Flask sets up a redirect package called :data:`flask.ext` where users
-should import the extensions from.  If you for instance have a package
-called ``flask_something`` users would import it as
-``flask.ext.something``.  This is done to transition from the old
-namespace packages.  See :ref:`ext-import-transition` for more details.
+their :file:`setup.py` files. If you for instance have a package called
+``flask_something`` users should import it as ``flask_something`` rather than
+using the soon to deprecated format ``flask.ext.something``. See
```

Comment:
This phrasing is slightly incorrect. The other changes are fine (and necessary for 1.0).